OpenRA.Mods.Common/UpdateRules/Rules/ChangeIntensityToDuration.cs

```diff
@@ -30,13 +30,7 @@ public override string Description
 		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
 		{
 			foreach (var sod in actorNode.ChildrenMatching("ShakeOnDeath"))
-			{
-				var intensity = sod.LastChildMatching("Intensity");
-				if (intensity == null)
-					continue;
-
-				intensity.RenameKeyPreservingSuffix("Duration");
-			}
+				sod.RenameLastChildMatching("Intensity", "Duration");
```

Comment:
When renaming nodes we really should be doing everything, not just the last one.  Otherwise the update will leave  you in a broken state if you're renaming a trait node and an actor has multiple.